### ISSUE TITLE
defaultIssuer and defaultName are internal and cannot be referenced

### ISSUE THREAD

**Issue by martindaum** [mattrubin/OneTimePassword#152]
I get the following errors when building my application:

In Token.swift:
Static let 'defaultIssuer' is internal and cannot be referenced from a default argument value
Static let 'defaultName' is internal and cannot be referenced from a default argument value

I have added OneTimePassword with cocoapods and my application uses Swift4.

**Comment by gabornagyfarkas** [comment]
We're having the same issue. With Swift 3.2 it compiles perfectly, so the problem most probably is Swift 4.

**Comment by gabornagyfarkas** [comment]
@martindaum as I see the problem is fixed as shown [here](https://github.com/mattrubin/OneTimePassword/commit/d51fe321df7a88895033968aad8937831ce6df61#diff-b014bc40c2c243722b9b3596ab2fdfc8). 

My problem is that I cannot add the develop branch as a dependency to my own pod. Could you please create a tag from it @mattrubin ?

Thanks!

**Comment by kingmatusevich** [comment]
Fix this please

**Comment by alisolutions** [comment]
Fix it please.

**Comment by ko0f** [comment]
In the mean time, change the `Podfile` code to
```
pod 'OneTimePassword', :git => 'https://github.com/mattrubin/OneTimePassword'
```

**Comment by mattrubin** [comment]
The CocoaPods compile issue is fixed, and version [3.0.1](https://github.com/mattrubin/OneTimePassword/releases/tag/3.0.1) of the pod is now available.

The underlying issue was CocoaPods not respecting the project's `.swift-version` file, and defaulting to Swift 4.0 for all pods. The `swift_version` podspec parameter added in CocoaPods 1.4.0 allows the podspec to specify a Swift version that will be used by the generated Pods project.
